vslib/src/VirtualSwitchSaiInterface.cpp

```diff
@@ -659,6 +659,12 @@ sai_status_t VirtualSwitchSaiInterface::create(
             return SAI_STATUS_FAILURE;
         }
 
+        // Initialize switch for VOQ attributes
+        if ((ss->initialize_voq_switch_objects(attr_count, attr_list)) != SAI_STATUS_SUCCESS)
```

Comment:
this is in the wrong place, it should be inside SwitchStateBase.cpp class in initialize objects

Reply:
As discussed in PR #701, this is unchanged.

Reply:
agree,  but there are places init_default_objects inside _init_switch and warm_boot_initialize_objects,
 current implementation you want to execute only for default objects, and this place here is actually executing it for both, which will cause to duplicate system port existence in virtual switch for warm boot, 
so i would propose to add attr_count/list to init_switch and move this initialization in line 604

Reply:
also when you will want to support warm boot scenario for this, then you will need to add warm boot case in init_switch also

Reply:
For comment 1: o.k. I fix proposed. 
For comment 2: o.k. when we support warm boot (only when system ports become dynamic config. Not sure when), this place will be used.

Reply:
Fixed. Moved initialize_voq_switch_objects() to init_switch() as proposed.